samples/net/mqtt_sn_publisher/README.rst

```diff
@@ -40,13 +40,29 @@ Requirements
 Build and Running
 *****************
 
-Currently, this sample application only supports static IP addresses.
+This sample application supports both static IP addresses and the Gateway Discovery process.
 Open the :file:`prj.conf` file and set the IP addresses according
-to the LAN environment.
+to the LAN environment. CONFIG_NET_SAMPLE_MQTT_SN_STATIC_GATEWAY can be used to select the
+static IP or Gateway discovery process.
 
-You will also need to start an MQTT-SN gateway. With Paho, you can either
-build it from source - see `PAHO MQTT-SN Gateway`_ - or run an unofficial
-docker image, like `kyberpunk/paho`_.
+You will also need to start an MQTT-SN gateway. A convenience Docker Compose specification file
+is provided in :file:`compose/docker-compose.yaml`. First, Start the net-tools configuration from
```

Comment:
typo in file name (you may actually want to call it docker-compose.yaml instead of just compose.yaml?), + use zephyr_file for it to be a proper link (and, incidentally, check that the referenced file actually exists :))

```suggestion
is provided in :zephyr_file:`samples/net/mqtt_sn_publisher/compose/compose.yaml`. First, Start the net-tools configuration from
```

Reply:
Added the `zephyr_file` change. Looked it up and `compose.yaml` is the recommended file name by Docker so leaving it at that :)